osmoutils/accum/accum_test.go

```diff
@@ -1418,18 +1472,6 @@ func (suite *AccumTestSuite) TestSetPositionCustomAcc() {
 			positionName:           validPositionName,
 			customAccumulatorValue: initialCoinsDenomOne,
 		},
-		"invalid update smaller than the initial value": {
```

Comment:
Note to reviewer: We no longer have this validation which is why these expectedErrors never happen